packages/venia-ui/i18n/en_US.json

```diff
@@ -109,9 +109,16 @@
     "navHeader.signInText": "Sign In",
     "navigationTrigger.ariaLabel": "Toggle navigation panel",
     "Order Status": "Order Status",
+    "orderDetails.billingInformationLabel": "Billing Information",
+    "orderDetails.buyAgain": "Buy Again",
+    "orderDetails.paymentMethodLabel": "Payment Method",
+    "orderDetails.printLabel": "Print Receipt",
+    "orderDetails.returnThis": "Return This",
+    "orderDetails.shippingInformationLabel": "Shipping Information",
+    "orderDetails.shippingMethodLabel": "Shipping Method",
+    "orderProgressBar.deliveredText": "Delivered",
```

Comment:
Weird merge? This was previously in alphabetical order but got moved above `orderHistoryPage` strings.

Reply:
Yeah, addressed the order.